### DISCUSSION TITLE
Sprint 2 Features

### DISCUSSION THREAD

**Discussion by raghusaripalli** [raghusaripalli/GatorGiveaway#29] [General]
For Sprint 2 some features which we need to work on:

- Enable images of products posted
- Use a CSS framework (Discussion Link [here](https://github.com/raghusaripalli/GatorGiveaway/discussions/28))
- Add user email, input field data validations and user email and password update.

Please post the features which you think needed to be included.

**Comment by tejasri555** [comment]
We also have to build APIs to read sellers' post and show it to buyers.

**Comment by ypalnati** [comment]
We can also have "Add to cart" feature for buyers.
